examples/nodejs-simple/README.md

```diff
@@ -39,71 +42,98 @@ docker run --network=host gcr.io/agones-images/nodejs-simple-server:0.1
-Health ping sent
-Health ping sent
+        name: local
+        state: Ready
+        labels: agones.dev/sdk-test-label,test-value & islocal,true
+        annotations: annotation,true
 Health ping sent
+Running for 20 seconds
+Setting an annotation
+GameServer Update:   
+        name: local
+        state: Ready
+        labels: agones.dev/sdk-test-label,test-value & islocal,true
+        annotations: agones.dev/sdk-test-annotation,test value & annotation,true
+Running for 30 seconds
+Marking server as ready...
+GameServer Update:
+        name: local
+        state: Ready
+        labels: agones.dev/sdk-test-label,test-value & islocal,true
+        annotations: agones.dev/sdk-test-annotation,test value & annotation,true
 Health ping sent
+Running for 40 seconds
+Allocating
+GameServer Update:
+        name: local
+        state: Allocated
+        labels: agones.dev/sdk-test-label,test-value & islocal,true
+        annotations: agones.dev/sdk-test-annotation,test value & annotation,true
+Running for 50 seconds
+Reserving for 10 seconds
+GameServer Update:
+        name: local
+        state: Reserved
+        labels: agones.dev/sdk-test-label,test-value & islocal,true
+        annotations: agones.dev/sdk-test-annotation,test value & annotation,true
 Health ping sent
+Running for 60 seconds
+GameServer Update:
+        name: local
+        state: Ready
+        labels: agones.dev/sdk-test-label,test-value & islocal,true
+        annotations: agones.dev/sdk-test-annotation,test value & annotation,true
+Running for 70 seconds
 Shutting down after 60 seconds...
-...marked for Shutdown
-Running for 60 seconds!
-Health ping sent
-Health ping sent
-Health ping sent
-Health ping sent
-Health ping sent
-Running for 70 seconds!
-Health ping sent
-Health ping sent
-Health ping sent
-Health ping sent
-Health ping sent
-Running for 80 seconds!
 Health ping sent
+Running for 80 seconds
+Running for 90 seconds
 Health ping sent
+Running for 100 seconds
+Running for 110 seconds
 Health ping sent
-Health ping sent
-Health ping sent
-Running for 90 seconds!
+Running for 120 seconds
+Running for 130 seconds
+Shutting down...
+GameServer Update:
+        name: local
+        state: Shutdown
+        labels: agones.dev/sdk-test-label,test-value & islocal,true
+        annotations: agones.dev/sdk-test-annotation,test value & annotation,true
+Health ping sent
+Running for 140 seconds
+Closing connection to SDK server
+Running for 150 seconds
+Exiting
+
+```
+
+You can optionally specify how long the server will stay up once the basic tests are complete.
+To do this pass arguments through, e.g. to increase the shutdown duration to 120 seconds:
+```
+$ make args="120" run
+```
+
+Please note that there is a max sleep time in Node.js of 2,147,483 seconds (~24 days). Values will be capped at this and negative values or zero will also be set to this.
+So to keep the server alive for a long time you could use the following:
+```
+$ make args="0" run
```

Comment:
I would make these (no pun intended :smile: ) the full Docker run command.

That way people don't need to pull down the full example source code -- and can also replicated within their Agones install as well.

Reply:
Ah sure. I was just basing this on the previous instructions. I usually run with `npm start` anyway.
If there's a need for both should we have both ways of running?
@roberthbailey you wrote the original README - any thoughts on this?

Reply:
Ah, if it's been this way for a while, I'll be less opinionated on changing to Docker run... but it still would be nice I think?